### DISCUSSION TITLE
Welcome to onedrive Discussions!

### DISCUSSION THREAD

**Discussion by abraunegg** [abraunegg/onedrive#1242] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yannisdran** [comment]
Hi there! Discussions was an important feature which was missing imho and it is good that it is set up. Hope that it will push the project and the community further.
I am a student and I use OneDrive for Business as provided from my university. However, I don't like win10 at all and after years of hesitation, I finally landed on Ubuntu 20.04 (installed on an old pc btw). I was able to set everything I need for daily work except OneDrive and a web camera (actually, an old android cellphone serving as a web camera).
See you around

**Comment by alicealysia** [comment]
Thank you so, so much for making this!
Recently setup a very odd, gaming windows server + linux server with gui and vnc so that I could have the best of both worlds running simultaneously, and I have to say, this was exactly what I needed to keep all of my files happy :+1:
